resample/bootstrap.py

```diff
@@ -175,6 +175,87 @@ def confidence_interval(
     )
 
 
+def bias(fn: Callable, sample: Sequence, **kwds) -> np.ndarray:
+    """
+    Calculate bias of the function estimate with the bootstrap.
+
+    Parameters
+    ----------
+    fn : callable
+        Function to be bootstrapped.
+    sample : array-like
+        Original sample.
+    **kwds
+        Keyword arguments forwarded to :func:`resample`.
+
+    Notes
+    -----
+    The bootstrap method cannot correct biases of linear estimators which are exactly
+    of order 1/N, where N is the number of i.i.d. observations, the jackknife should be
+    used for those. For bias estimation, the balanced bootstrap is strongly
+    recommended. The ordinary bootstrap may requires a much larger number of replicas to
+    produce a similarly accurate result.
+
+    Returns
+    -------
+    ndarray
+        Bootstrap estimate of bias (= expectation of estimator - true value).
+    """
+    theta = fn(sample)
+    thetas = bootstrap(fn, sample, **kwds)
+    return np.mean(thetas, axis=0) - theta
+
+
+def bias_corrected(fn: Callable, sample: Sequence, **kwds) -> np.ndarray:
+    """
+    Calculates bias-corrected estimate of the function with the bootstrap.
```

Comment:
```suggestion
    Calculate bias-corrected estimate of the function with the bootstrap.
```
Tiny nitpick to have imperative mood like other docstrings